packages/code-du-travail-frontend/src/common/Mdx.ts

```diff
@@ -7,7 +7,7 @@ import unified from "unified";
 
 type Props = {
   markdown: string;
-  components?: any[];
+  components: any;
 };
 
 const Mdx: React.FC<Props> = ({ markdown, components = {} }) => {
```

Comment:
```suggestion
const Mdx: React.FC<Props> = ({ markdown, components }) => {
```

La valeur par défaut n'est plus utile du coup :)